runtime/src/gpu/amd/gpu-amd.c

```diff
@@ -128,29 +128,17 @@ static void chpl_gpu_impl_set_globals(hipModule_t module) {
 }
 
 
-void chpl_gpu_impl_init() {
-  int         num_devices;
-
-  CHPL_GPU_DEBUG("Initializing GPU layer.\n");
-  CHPL_GPU_DEBUG("  Memory allocation strategy for ---\n");
-  #ifdef CHPL_GPU_MEM_STRATEGY_ARRAY_ON_DEVICE
-    CHPL_GPU_DEBUG("    array data: device memory\n");
-    CHPL_GPU_DEBUG("         other: page-locked host memory\n");
-  #else
-    CHPL_GPU_DEBUG("    array data: unified memory\n");
-    CHPL_GPU_DEBUG("         other: unified memory\n");
-  #endif
-
-  // ROCM initialization
+void chpl_gpu_impl_init(int* num_devices) {
```

Comment:
Thanks for cleaning on the AMD runtime and getting the `CHPL_GPU_DEBUG` lines out (since this is all handled by the vendor-neutral layer now right?).

Reply:
Right.